server/bin/pbench-backup-tarballs.py

```diff
@@ -179,44 +180,42 @@ def backup_to_local(
         return Status.FAIL
 
     # Check if tarball exists in local backup
-    backup_tar = os.path.join(backup_controller_path, resultname)
-    if os.path.exists(backup_tar) and os.path.isfile(backup_tar):
-        backup_md5 = os.path.join(backup_controller_path, "{}.md5".format(resultname))
+    backup_tar = backup_controller_path / resultname
+    if backup_tar.exists() and backup_tar.is_file():
+        backup_md5 = backup_controller_path / "{}.md5".format(resultname)
 
         # check backup md5 file exist and it is a regular file
-        if os.path.exists(backup_md5) and os.path.isfile(backup_md5):
+        if backup_md5.exists() and backup_md5.is_file():
             pass
         else:
             # backup md5 file does not exist or it is not a regular file
             logger.error(
-                "{md5} does not exist or it is not a regular file".format(
-                    md5=backup_md5
-                )
+                "{} does not exist or it is not a regular file".format(backup_md5)
```

Comment:
This is an example where because it is logging, we should fix this to be:

```
logging.error("{} does not exist or it is not a regular file", backup_md5)
```

Reply:
I changed it into this as it must have been fixed in other PR and is not reflected here due to master and br69 difference. It was the error which lint was facing so I resolved it.

Reply:
Before we continue reviewing, let's rebase this code on br69 and then we'll continue the review.